boards/arm/frdm_k22f/frdm_k22f.yaml

```diff
@@ -0,0 +1,17 @@
+identifier: frdm_k22f
+name: NXP FRDM-K22F
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+supported:
+  - adc
+  - i2c
+  - nvs
+  - spi
+  - gpio
+  - usb_device
+  - watchdog
```

Comment:
Have you tested all of these functions?

Reply:
Will test and report :-)

Reply:
I got the same failure.

Reply:
Great (or not) :-)
Not great for frdm_k64 either...
https://paste.pound-python.org/show/d7wTVrBF7ZhhKJjfWA7i/